CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -464,6 +464,10 @@ Each of these folders should hold the same image resized to a different resoluti
 
 The [rescaling script](https://github.com/matkoniecz/rescaling_for_android) may be useful, but you can also do this manually with Gimp or similar software.
 
+Please make sure that the images do not take too much disk space. On StretComplete images, JPEG quality of 60% usually looks exactly the same as 80% (or 90%) while being noticeably smaller, and sometimes reducing it down to even 20% will not degrade quality visibly! Play with settings to see which is the smallest size which does not degrade image quality visibly.
```

Comment:
are you sure about 60%? I was not adding images in some time, but I remember noticing visible quality damage at 80%, to say nothing about 60%.

That is why https://github.com/matkoniecz/rescaling_for_android/blob/master/rescale.rb is using 85%

Reply:
Well, it depends on exact image, so for automated tools its hard to guess :shrug: 

See this [image comparison](https://github.com/Helium314/SCEE/pull/644#issuecomment-2341964292), where `20%` (!!!) image quality (on the right) if pretty comparable to much higher JPEG quality **even when viewed on desktop sizes** - yes, you can say they're not exactly the same if you have them one near the other, but if anything, that `20%` JPEG might even look slightly better.   

When reduced to size of mobile phone screen, however, even those tiny differences become invisible.

That is however probably very hard to do without human in the loop, which is why I've tried to emphasize playing with settings to get best results. That `rescale.rb` might get configurable JPEG quality as a parameter though, to ease the trying.

Reply:
anyway, I've removed specific percentage recommendations now, and emphasized testing.

Reply:
Oh, I guess that depends on topic of image. I was testing with gravel and leaves which are likely far more fragile than simple mechanical object on monocolour background.